Planner/src/TaskDAO.java

```diff
@@ -0,0 +1,219 @@
+			Statement stmt = conn.createStatement();
+		      
+			// Step 3 & 4: Execute a SQL UPDATE via executeUpdate()
+			//   which returns an int indicating the number of rows affected.
+			String strUpdate = "update tasks set nameTask =' " + t.getNameTask() + "', descriptionTask= '" + t.getDescriptionTask() + "' where taskID=" + String.valueOf(t.getTaskID());
+			System.out.println("The SQL query is: " + strUpdate);  // Echo for debugging
+			int countUpdated = stmt.executeUpdate(strUpdate);
+			System.out.println(countUpdated + " records affected.");
+			
+			// Step 3 & 4: Issue a SELECT to check the UPDATE.
+			String strSelect = "select * from tasks" + " where taskID=" + String.valueOf(t.getTaskID());
+			System.out.println("The SQL query is: " + strSelect);  // Echo for debugging
+			ResultSet rset = stmt.executeQuery(strSelect);
+			
+			while(rset.next()) {   // Move the cursor to the next row
+				String nameTask = rset.getString("nameTask");
+	            String descriptionTask = rset.getString("descriptionTask");
+	            Date startTime = null;
+	            Date endTime = null;
+	            try {
+	            	startTime = new SimpleDateFormat("dd/MM/yyyy HH:mm").parse(rset.getString("startTime"));
+	            	endTime = new SimpleDateFormat("dd/MM/yyyy HH:mm").parse(rset.getString("endTime"));
+	            } catch (ParseException e) {
+	            }
+	            int assignedTo = rset.getInt("assignedTo");
+	            int taskID = rset.getInt("taskID");
+	            
+	            /*
+	            // Step 3 & 4: Issue a SELECT to check the UPDATE.
+				String selectUser = "select * from users" + " where ID=" + String.valueOf(assignedTo);
+				System.out.println("The SQL query is: " + strSelect);  // Echo for debugging
+				ResultSet rsetUser = stmt.executeQuery(selectUser);
+				User user = null;
+				while(rsetUser.next()) {   // Move the cursor to the next row
+					String name = rsetUser.getString("firstName");
+		            String surname = rsetUser.getString("surname");
+		            int id = rsetUser.getInt("ID");
+		            user = new User(name, surname, id);
+				}
+				*/
+	            task = new Task(nameTask, descriptionTask, startTime, endTime, null, taskID);
+			}
+		} catch(SQLException ex) {
+			ex.printStackTrace();
+		}
+		 return task;
+	}
+	
+	static Task insertTask(Task t, ArrayList<User> users) {
+		Task task = null;
+		try {
+			// Step 1: Allocate a database 'Connection' object
+			Connection conn = DriverManager.getConnection(
+			"jdbc:mysql://localhost:3306/planner?useSSL=false", "Justyna", "qwer1234"); // MySQL
+			
+			// Step 2: Allocate a 'Statement' object in the Connection
+			Statement stmt = conn.createStatement();
+			
+			// Step 3: Execute a SQL INSERT statement via executeUpdate(),
+			//   which returns an int indicating the number of rows affected.
+			// INSERT a record
+			String sqlInsert = "insert into tasks " // need a space
```

Comment:
You need to change this insert statement with the one that you tried in MySql Workbench and it worked:
"insert into tasks (nameTask, descriptionTask, start, end, assignedTo) values ('" + t.getNameTask() + "', '" + t.getDescriptionTask() + "', '" + t.getStartTime() + "', '" + t.getEndTime() +  "', " + t.getAssignedTo().getId() + ")";

Make sure that you put the right column names from mysql in insert into tasks in the column sections. 

Update the insert statement for the users as well.